packages/app/pages/course/[cid]/insights.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactElement } from "react";
+import { API } from "@koh/api-client";
+import useSWR from "swr";
+import { Tooltip, Card } from "antd";
+import { InfoCircleOutlined } from "@ant-design/icons";
+import { useRouter } from "next/router";
+import { StandardPageContainer } from "../../../components/common/PageContainer";
+import SimpleDisplayComponent from "../../../components/Insights/components/SimpleDisplayComponent";
+import SimpleChartComponent from "../../../components/Insights/components/SimpleChartComponent";
+
+export default function Insights(): ReactElement {
+  // TODO: In the future this will come from the users specific insights that want to see
+  const insights = ["totalQuestionsAsked", "totalStudents"];
+
+  return (
+    <>
+      <StandardPageContainer>
+        <h1 style={{ margin: "20px" }}>Insights Dashboard</h1>
+        <divider />
+        <div style={{ display: "flex", direction: "ltr" }}>
+          {insights?.map((insightName: string) => {
+            return (
+              <RenderInsight key={insightName} insightName={insightName} />
+            );
+          })}
+        </div>
+      </StandardPageContainer>
+    </>
+  );
+}
+
+interface RenderInsightProps {
+  insightName: string;
+}
+
+function RenderInsight({ insightName }: RenderInsightProps): ReactElement {
+  const router = useRouter();
+  const { cid } = router.query;
+
+  const { data: insight, error, mutate } = useSWR(
+    cid && `api/v1/insights/${cid}/${insightName}`,
+    async () => API.insights.get(Number(cid), insightName)
+  );
+
+  if (!insight) {
+    // TODO: Create loading shell
+    return null;
+  }
+
+  // Determine which insight component to render
+  let InsightComponent;
+  switch (insight.component) {
+    case "SimpleDisplay":
+      InsightComponent = SimpleDisplayComponent;
+      break;
+    case "SimpleChart":
+      InsightComponent = SimpleChartComponent;
+      break;
+    default:
+    // Return an error component
+    // Log error
+  }
+
+  return (
+    <Card
+      size="small"
```

Comment:
I'm worried that there are insights that aren't going to fit into this specific Insight size -- what's the plan in that case? For instance, things that require searching/scrolling through lists (TA Check In/Check Out is an example -- though that could also probably fit into this card size)

Reply:
My thinking was that we could set the size on the InsightComponent instead of the Card and which would allow the card to expand to whatever size the InsightComponent needs. 

Tbh I haven't fully thought through how all the styling is going to work. Hoping to talk with @liustanley about that soon.

Reply:
I think that'd be a great idea -- I'm down to add those to a PR after thsi one is merged and see what people think